packages/synapse-react-client/src/components/InteractiveCopyIdsIcon.tsx

```diff
@@ -1,26 +1,26 @@
 import React from 'react'
-import { IconButton, Tooltip } from '@mui/material'
+import { IconButton, IconButtonProps, Tooltip } from '@mui/material'
 import { TOOLTIP_DELAY_SHOW } from './SynapseTable/SynapseTableConstants'
 import IconSvg from './IconSvg/IconSvg'
 
-export type InteractiveCopyIdsIconProps = {
+export type InteractiveCopyIdsIconProps = Omit<IconButtonProps, 'onClick'> & {
   onCopy: () => void
 }
 
 export const InteractiveCopyIdsIcon = (props: InteractiveCopyIdsIconProps) => {
-  const { onCopy } = props
+  const { onCopy, ...buttonProps } = props
   return (
     <Tooltip
       title="Copy IDs to the clipboard"
       enterNextDelay={TOOLTIP_DELAY_SHOW}
       placement="right"
     >
       <IconButton
-        sx={{ height: '40px' }}
         data-testid="copySynIdsButton"
+        {...buttonProps}
         onClick={onCopy}
       >
-        <IconSvg icon="contentCopy" />
+        <IconSvg wrap={false} icon="contentCopy" fontSize={'inherit'} />
```

Comment:
`fontSize="inherit"` is how the MUI docs instruct to have an SvgIcon respect the size of the IconButton it is contained within.